examples/tv-casting-app/darwin/TvCasting/MatterBridge/CastingServerBridge.mm

```diff
@@ -153,4 +154,39 @@ - (void)sendUserDirectedCommissioningRequest:(NSString * _Nonnull)commissionerIp
         });
     });
 }
+
+- (void)openBasicCommissioningWindow:(nullable void (^)(bool))commissioningCompleteCallback
+                         clientQueue:(dispatch_queue_t _Nonnull)clientQueue
+    commisisoningWindowOpenedHandler:(nullable void (^)(bool))commisisoningWindowOpenedHandler
+{
+    ChipLogProgress(AppServer, "CastingServerBridge().openBasicCommissioningWindow() called");
+
+    dispatch_async(_chipWorkQueue, ^{
+        CHIP_ERROR err = CastingServer::GetInstance()->OpenBasicCommissioningWindow(
+            [&commissioningCompleteCallback](CHIP_ERROR err) { commissioningCompleteCallback(CHIP_NO_ERROR == err); });
+
+        dispatch_async(clientQueue, ^{
+            commisisoningWindowOpenedHandler(CHIP_NO_ERROR == err);
```

Comment:
So this will call the callback before the commissioning window is actually opened, possibly, right?  The API should probably be clearly documented in terms of what it does.

Reply:
Updated its name to commissioningWindowRequestedHandler to indicate that the opening of a commissioning window has only been requested. 
Also, added documentation for this method and the others in CastingServerBridge.h